config/clusters/2i2c.cluster.yaml

```diff
@@ -131,6 +131,8 @@ hubs:
         hub:
           config:
             Authenticator:
+              # Only usernames matching this regex will be allowed to login into the hub
+              # ref: https://jupyterhub.readthedocs.io/en/stable/api/auth.html#jupyterhub.auth.Authenticator.username_pattern
```

Comment:
```suggestion
              # We do not define allowed_users here since only usernames matching this regex
              # will be allowed to login into the hub. Ref: https://jupyterhub.readthedocs.io/en/stable/api/auth.html#jupyterhub.auth.Authenticator.username_pattern
```